### PR TITLE
Fixed Doc Tickets - DOC-3527 and DOC-4650

### PR DESCRIPTION
**Pull request by ghost**
Fixed the following doc tickets:

- DOC-3527
- DOC-4650

### CODE REVIEW

File: modules/cli/pages/cbft-bleve-moss.adoc [hunk 1]
```diff
@@ -0,0 +1,521 @@
+= cbft-bleve for Moss Indexes
+:description: pass:q[The `cbft-bleve` tool supports commands that return information on _Moss_ index partitions.]
+:page-topic-type: reference
```
Comment (simon-dew) [inline modules/cli/pages/cbft-bleve-moss.adoc:3]:
```suggestion
:page-topic-type: reference
:page-aliases: cbft-bleve-dump
```

**Review by simon-dew — CHANGES_REQUESTED** [review]
Need page alias for deleted `cbft-bleve-dump` page.

File: modules/cli/pages/cbft-bleve-moss.adoc (hunk 1 above)
Comment (ghost) [reply modules/cli/pages/cbft-bleve-moss.adoc:3]: Hi @simon-dew the page alias "cbft-bleve-dump" that you mentioned is actually added to the new cbft-bleve.adoc file. This is the renamed file. So, there is no need to add that page alias to "cbft-bleve-moss.adoc" file.

Comment (simon-dew) [reply modules/cli/pages/cbft-bleve-moss.adoc:3]: Ah sorry, I missed that!

**Review by simon-dew — APPROVED** [review]

**Pull request merged**